### ISSUE TITLE
Homepage: Update text in the "splash announcement"

### ISSUE THREAD

**Issue by smarziano** [smartchicago/chicago-early-learning#756]
Related: #753 

We are looking to update the content on the homepage "splash announcement" section:
<img width="1277" alt="screen shot 2017-05-11 at 2 37 13 pm" src="https://cloud.githubusercontent.com/assets/5550969/25968510/5c42e726-3657-11e7-9e64-aab18b4b3e22.png">

**Updated content**
There is no limit to what your child can accomplish! Jumpstart your child's education with Chicago Early Learning!

[Complete your child's preschool application online today!](https://cys.mycopa.com/familyPortal/welcome.epl) 
_If your child qualifies, you will receive an email on June 1st with information on how to verify your child’s assignment._

Questions? See our [FAQs page](https://chicagoearlylearning.org/faq) for more information, or call the Chicago Early Learning Hotline: 312-229-1690

**Comment by smarziano** [comment]
Spanish translation from "Complete your child's preschool application..."
¡Complete una aplicación para su niño hoy para la escuela preescolar! [link]
_Si califica su niño, reciba un correo electrónico el 1º de junio con información en cómo verificar la asignación de su niño._

¿Preguntas? Visite nuestra pagina de Preguntas y Respuestas [link] para más información, o llame al Chicago Early Learning línea de acceso directo: 312-229-1690

I am waiting for the translation for the first part.

**Comment by smarziano** [comment]
English version is up on staging, so I am going to add the "Waiting on merge" label.

Waiting for Spanish translation for "There is no limit to what your child can accomplish! Jumpstart your child's education with Chicago Early Learning!"

**Comment by smarziano** [comment]
Will create a new issue for Spanish translation. Closing this issue.
